src/dlgPackageExporter.cpp

```diff
@@ -302,7 +302,16 @@ void dlgPackageExporter::slot_export_package()
         return;
     }
 
-    close();
+    showUploadNudge();
+
+}
+
+void dlgPackageExporter::showUploadNudge()
+{
+   ui->infoLabel->setText(R"(<a href="https://forums.mudlet.org/posting.php?mode=post&f=6">Upload package to Mudlet</a>)");
```

Comment:
"to Mudlet" what? Add "forums", "community", "for other users", etc. something like that. Maybe also talk about "share" not "upload"
--> "Share package with Mudlet community"
Also maybe add "Export successful!" message here, before linking to the forums.

Reply:
Don't need to explain the what down to 100% detail I don't think. It's obvious once you click that link.